core/pom.xml

```diff
@@ -419,7 +419,7 @@ THE SOFTWARE.
     <dependency><!-- groovy shell uses this but it doesn't declare this dependency -->
       <groupId>org.fusesource.jansi</groupId>
       <artifactId>jansi</artifactId>
-      <version>1.11</version>
+      <version>1.14</version>
```

Comment:
@recena Indeed, unrelated, even if it would not be a bad idea to upgrade it but clearly no need to include it here. I will mark it as [wip] as I still want to test it more, I may need or want to include few more layout fixes to improve the UX.